sites/joshuakeel.com.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Joshua Keel'
+url: 'https://joshuakeel.com'
+tags: ['writer', 'web developer', 'philosophy', 'ethics', 'godless heathen']
```

Comment:
> godless heathen

go to every-layout.dev and view source ;)

Reply:
Oooh, nice easter egg. ;)

Reply:
😎